Scripts/Editor/Utility/WitUnderstandingViewer.cs

```diff
@@ -53,6 +54,26 @@ static void Init()
             window.Show();
         }
 
+        private void OnSelectionChange()
+        {
+            if (Selection.activeGameObject)
+            {
+                var wit = Selection.activeGameObject.GetComponent<Wit>();
+                if (wit)
+                {
+                    wit.events.OnResponse.AddListener((r) =>
+                    {
+                        response = r;
+                        var u = r["text"].Value;
+                        if (!string.IsNullOrEmpty(u))
+                        {
+                            utterance = u;
+                        }
```

Comment:
Could get rid of single-variable names here by doing

```
if (!string.IsNullOrEmpty(response["text"].Value))
{
    utterance = response["text"].Value;
}
```

although that duplicates the unwrapping.

Reply:
Yeah, that duplicates the lookup for the value for text. Maybe I should add a TryGetValue to the node.